yarn.lock

```diff
@@ -1591,6 +1591,16 @@ ajv@^6.10.0, ajv@^6.12.4:
     json-schema-traverse "^0.4.1"
     uri-js "^4.2.2"
 
+ajv@^6.5.5:
```

Comment:
Not sure why the yarn.lock has any changes? Could you please revert these if possible. Thanks.

Reply:
reverted already!